projects/rero/ng-core/src/lib/record/export-button/export-button.component.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * RERO angular core
+ * Copyright (C) 2022 RERO
+ * Copyright (C) 2022 UCLouvain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import {Component, Input} from '@angular/core';
+
+
+@Component({
+  selector: 'ng-core-export-button',
+  templateUrl: './export-button.component.html'
+})
+export class ExportButtonComponent {
+
+  /** Export formats configuration */
+  @Input()
+  exportOptions: {
```

Comment:
single line ? ==> `@Input() exportOptions: { ... }`
This notation is used in all UI project and in many ng-core component (or it should be !)

Reply:
ok